.github/workflows/on-pr-merge.yml

```diff
@@ -3,27 +3,49 @@ on:
   workflow_dispatch: {}
   push:
     branches:
-      - main
+      - mm-PF-569-debug-bumper-gha # TODO: change this back to main before merge
 
 jobs:
   bump-version-publish-release:
     runs-on: ubuntu-latest
     if: "!contains( github.event.sender.login, 'broadbot')"
     steps:
       - name: Checkout current code
-        uses: actions/checkout@master
+        id: checkout_code
+        uses: actions/checkout@v2
         with:
-          token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+          ref: mm-PF-569-debug-bumper-gha # TODO: change this back to main before merge
       - name: Bump tag and build version
+        id: bump_tag
         uses: databiosphere/github-actions/actions/bumper@v0.0.3
         env:
           GITHUB_TOKEN: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
-          RELEASE_BRANCHES: main
+          DEFAULT_BUMP: minor
+          RELEASE_BRANCHES: mm-PF-569-debug-bumper-gha # TODO: change this back to main before merge
           VERSION_FILE_PATH: build.gradle
           VERSION_LINE_MATCH: ^version
+      - name: Updating code to get the version bump changes
+        id: update_code_post_version_bump
+        run: |
+          git pull
+        env:
+          GITHUB_TOKEN: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+      - name: Render config
+        id: render_config
+        run: |
+          # this step does the equivalent of the tools/render-config.sh script
+          # on local machines, the script fetches a SA from Vault
+          # in GH actions, the SA key is stored in a GH repo secret
+          # regardless of how it was fetched, the publish-release script expects this
+          # key to be stored in rendered/ci-account.json
+          mkdir -p rendered
```

Comment:
Is there any concern about leaving `rendered/ci-account.json` hanging around at the end of the workflow run?

Reply:
No, the GH action runners won't be reused as is.